### DISCUSSION TITLE
Pls tell me

### DISCUSSION THREAD

**Discussion by HizunaBT** [skyline-emu/skyline#1809] [Q&A]
Can someone give me product keys and title keys for skyline emu V.0.0.3?
Im not have product and title keys..
and can anyone tell me what games are supported on skyline emulator in the latest version?

**Comment by guranu** [comment]
Get prod.keys and title.keys from you're switch, piracy isn't allowed here. Also check https://github.com/skyline-emu/skyline-games-list to see wich game is playable.

**Reply by HizunaBT** [reply]
So if im not have nintendo switch control i cant get product keys and title keys?

**Reply by Crytonics** [reply]
You can't emulate game if you don't have switch because you need to dump game and keys from your switch.

**Reply by HizunaBT** [reply]
Owh ok bro..thanks for the information

**Reply by Mikatakhan** [reply]
You dont have to have a switch to play switch games on skyline. Just download roms from google. And search on youtube for which games are playable.

**Reply by s3lverd** [reply]
> You dont have to have a switch to play switch games on skyline. Just download roms from google. And search on youtube for which games are playable.

Thats piracy and we DO NOT support that.

**Reply by dima-xd** [reply]
> You dont have to have a switch to play switch games on skyline. Just download roms from google. And search on youtube for which games are playable.

Just go to jail

**Reply by guranu** [reply]
> You dont have to have a switch to play switch games on skyline. Just download roms from google. And search on youtube for which games are playable.

Are you stupid or something piracy isn't allowed here.

**Reply by parknich** [reply]
> You dont have to have a switch to play switch games on skyline. Just download roms from google. And search on youtube for which games are playable.

myu guy what thats illegal

**Reply by Edrokski777** [reply]
Serious question what if u dumped your game and keys but no longer own a switch. But still own the game?

**Reply by guranu** [reply]
I guess that's technically legal since you you bought the games and dumped them from the switch.
